packages/terra-tabs/src/Tabs.jsx

```diff
@@ -131,16 +151,37 @@ class Tabs extends React.Component {
           showIcon={child.props.showIcon}
           render={() => tabContent}
           isDisabled={child.props.isDisabled}
+          isClosable={isClosable}
           variant="framework"
         />,
       );
     });
+    const handleTabsStateChange = (newValue, itemKey, event) => {
+      if (newValue.length > 0) {
+        let activeAfterClosed = '';
```

Comment:
can we rename this to activetab ?

Reply:
Yes Sure, Done  https://github.com/cerner/terra-framework/commit/8a5e358205a6f99d85acb80396bd87bb50bda2e8